src/huggingface_hub/_commit_api.py

```diff
@@ -173,7 +173,6 @@ def upload_lfs_files(
     repo_type: str,
     repo_id: str,
     token: str,
-    revision: str,
```

Comment:
What was the use of this?

If we remove it, we would break backwards compatibility. I would add a deprecation warning that it will be removed in 0.11

Reply:
This particular method is private so I think we can change it however we want (am I correct?)

Reply:
I didn't realize this was internal, then it SG!